java/com/google/copybara/Core.java

```diff
@@ -1166,7 +1166,7 @@ public TodoReplace todoReplace(
       },
       useStarlarkThread = true)
   @DocDefault(field = "paths", value = "glob([\"**\"])")
-  @DocDefault(field = "reverse", value = "`regex`")
+  @DocDefault(field = "reverse", value = "regex")
```

Comment:
Could you also run the generate_reference rule and add the resulting reference.md to the change?

TY!